src/libmongoc/tests/test-mongoc-read-prefs.c

```diff
@@ -866,32 +872,28 @@ test_read_prefs_mongos_hedged_reads (void)
 
    mongoc_read_prefs_set_hedge (prefs, &hedge_doc);
 
-   /* exhaust cursor is required so the driver downgrades the OP_QUERY find
-    * command to an OP_QUERY legacy find */
```

Comment:
Since the new behavior only affects 5.1+ servers, can this test change be reverted?

The mock server in this test reports wire version 9 (corresponding to 4.4 server version).

Reply:
Will revert.